daemon/images/image.go

```diff
@@ -24,11 +24,11 @@ import (
 
 // ErrImageDoesNotExist is error returned when no image can be found for a reference.
 type ErrImageDoesNotExist struct {
-	ref reference.Reference
+	Ref reference.Reference
 }
```

Comment:
For a follow-up; we need to check if we really need this error-type, as it already implements `errdefs.NotFound`, perhaps we don't need it (with the possible exception of the "special" `No such ...` prefix, which may be needed for older CLIs).

Reply:
Yeah this one might not be needed, the only difference is that this error type formats the message with `"No such image: %s", ref` whereas the `errdefs.NotFound` doesn't have that extra data